bundles/org.openhab.binding.goveelan/src/main/resources/OH-INF/i18n/goveelan.properties

```diff
@@ -0,0 +1,85 @@
+# add-on
+
+addon.goveelan.name = GoveeLan Binding
+addon.goveelan.description = This is the binding for handling Govee Lights via the LAN-API interface.
+
+# thing types
+
+thing-type.goveelan.govee-light.label = Govee Light Thing
+thing-type.goveelan.govee-light.description = Govee Light controllable via LAN API
+
+# thing types config
+
+thing-type.config.goveelan.govee-light.refreshInterval.label = Light refresh interval (sec)
+thing-type.config.goveelan.govee-light.refreshInterval.description = The amount of time that passes until the device is refreshed
+
+# channel types
+
+channel-type.goveelan.colortemperatureabs.label = Color Temperature (Absolute)
```

Comment:
Looks like this file needs to be regenerated, but let's get back to that.

Reply:
I guess it's the only one atm the needed to be adapted. I need to change file anyway after refactoring from goveelan to govee.

Reply:
The channel types for absolute color temperature seems to be missing now with the latest commits?